package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@chakra-ui/icons": "^1.1.1"
```

Comment:
I dont think this file is needed

Reply:
Don't we use the AddIcon and DeleteIcon from @chakra-ui/icons?

Reply:
Yes but its already installed just version 1.2.1 (see yarn.lock file)

Reply:
Alright, will remove it